builtin/providers/aws/resource_aws_opsworks_instance.go

```diff
@@ -886,6 +895,16 @@ func resourceAwsOpsworksInstanceDelete(d *schema.ResourceData, meta interface{})
 	return nil
 }
 
+func resourceAwsOpsworksInstanceImport(
+	d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+	// Neither delete_eip nor delete_ebs can be fetched
+	// from any API call, so we need to default to the values
+	// we set in the schema by default
+	d.Set("delete_ebs", true)
+	d.Set("delete_eip", true)
```

Comment:
These are not returned from the API and are only set in our statefile so we set the defaults on import